index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+
+<html Lang="en-GB">
+
+<head>
+	
```

Comment:
OPTIONAL: using normalize.css might be a good way to prevent any potential cross browser bugs.